packages/smooth_app/lib/generic_lib/buttons/smooth_large_button_with_icon.dart

```diff
@@ -42,7 +40,7 @@ class SmoothLargeButtonWithIcon extends StatelessWidget {
 
     return SmoothSimpleButton(
       minWidth: double.infinity,
-      padding: padding ?? const EdgeInsets.all(10),
+      padding: padding ?? const EdgeInsets.symmetric(vertical: 10),
```

Comment:
By changing this button, it will have an impact on the whole app.
Please provide specific values when you need it.

Reply:
Actually in figma design those buttons' horizontal padding was slightly high that is why i made this change.
However i will revert back to the original, no issue.